docker/share/pom.xml

```diff
@@ -9,14 +9,14 @@
     <name>Alfresco DBP Share Packaging</name>
 
     <properties>
-        <alfresco.rm.share.version>2.6.0</alfresco.rm.share.version>
+        <alfresco.gs.share.version>3.1.0-SNAPSHOT</alfresco.gs.share.version>
```

Comment:
Aren't we missing `alfresco-saml-share`?

Reply:
Ops! I missed that! Thanks for spotting it!